discover/portal/articles/210-setting-up/02-instance-settings/02-instance-settings.markdown

```diff
@@ -0,0 +1,217 @@
+    manually.
+
+-   Enable/Disable forgotten password functionality.
+
+-   Enable/Disable request password reset links.
+
+-   Enable/Disable account creation by strangers. If running an Internet
+    site, leave this enabled so visitors can create accounts on your site.
+
+-   Enable/Disable account creation by those using an email address in the
+    domain of the company running the site (which is set on the General page of
+    Instance Settings). This is handy if you're using Liferay to host both
+    internal and external web sites. Make sure all internal IDs are created by
+    administrators but external Users can register for IDs themselves.
+
+-   Enable/Disable email address verification. If enabled,
+    Users receive a verification email with a link back to the virtual instance,
+    verifying that the email address they entered is valid.
+
+By default, all settings except for the last are enabled. User authentication by
+email address is an important default for the following reasons:
+
+1.  An email address is unique to the User who owns it.
+
+2.  People remember their email addresses. A Users who hasn't logged in for a
+    while could forget their screen name.
+
+3.  If email address isn't used to authenticate, a User might change her email
+    address then forget to update the email address in her profile. If this
+    occurs, no notifications sent by the virtual instance will reach the User.
+    Keeping the email address at the forefront of a User's mind when she logs in
+    helps ensure the User keeps it current. 
+
+To leanr to configure a third party authentication service or set up Single Sign
+On (SSO), see the 
+[security documentation](/discover/deployment/-/knowledge_base/7-1/liferay-portal-security).
+
+## Users [](id=Users)
+
+The *Users* section has three tabs: Fields, Reserved Credentials and Default
+User Associations.
+
+In the Fields tab, enable/disable these fields on the Add/Edit User Form:
+
+- Autogeneration of screen names
+
+- Birthday field
+
+- Gender field
+
+In the Reserved Credentials tab, specify the screen names and email addresses
+Users aren't allowed to register with. Use this feature to prevent Users from
+creating IDs that look like administrative IDs or that have reserved words in
+their names.
+
+Default User Associations has four fields for listing (one per line) Sites,
+Organization Sites, Soles, and User Groups you want new Users to become members
+of automatically. By default, new Users are assigned to only the Users role. You
+can also choose whether to apply these configurations to existing Users.
+
+User groups are handy tools for pre-populating your Users' private sites with
+a certain page set, defined by a site template. If you've done that and want all
+Users you add to immediately be added to the User group and inherit those pages,
+add the User group here.
+
+## Terms of Use [](id=terms-of-use)
+
+Terms of Use are important when you need them, but not all sites do. This is
```

Comment:
We may want to add a note here that by default the terms of use are enabled but with a default text, so it's important to customize the text (through a web content) or disable them.

Reply:
Suggestion:

>Terms of Use are important when you need them, but not all sites do. Since
>they're enabled by default, one of your first actions should be to disable or
>replace the default, placeholder terms. This section of Instance Settings is
>where you disable the requirement for all Users to read the terms of use, and
>where you set the Group ID and Article ID for the Web Content Article you're
>using for your Terms of Use.

Reply:
👍